src/main/java/carpet/script/api/Inventories.java

```diff
@@ -19,14 +27,6 @@
 import carpet.script.value.StringValue;
 import carpet.script.value.Value;
 import carpet.script.value.ValueConversions;
-
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Objects;
-import java.util.Optional;
-import java.util.OptionalInt;
-import java.util.Set;
-
 import net.minecraft.core.HolderSet;
```

Comment:
why you edit this file? i cannot see the reason behind it. 
you dont need to commit all of your edited files

Reply:
Those file were added by mistake. I'm going to remove them